Marlin/language_an.h

```diff
@@ -59,21 +60,35 @@
 #define MSG_PREHEAT_ABS_ALL                 "Precalentar ABS a"
 #define MSG_PREHEAT_ABS_BEDONLY             "Prec. ABS Base"
 #define MSG_PREHEAT_ABS_SETTINGS            "Achustar tem. ABS"
+#define MSG_H1                              "1"
+#define MSG_H2                              "2"
+#define MSG_H3                              "3"
+#define MSG_H4                              "4"
 #define MSG_COOLDOWN                        "Enfriar"
 #define MSG_SWITCH_PS_ON                    "Enchegar Fuent"
 #define MSG_SWITCH_PS_OFF                   "Desenchegar Fuent"
 #define MSG_EXTRUDE                         "Extruir"
 #define MSG_RETRACT                         "Retraer"
 #define MSG_MOVE_AXIS                       "Mover Eixes"
+#define MSG_LEVEL_BED                       "Level bed"
 #define MSG_MOVE_X                          "Move X"
 #define MSG_MOVE_Y                          "Move Y"
 #define MSG_MOVE_Z                          "Move Z"
 #define MSG_MOVE_E                          "Extruder"
+#define MSG_MOVE_E1                         "1"
```

Comment:
The following directives were left out on propose, makes no sense to translate them:
`MSG_H1 MSG_H2 MSG_H3 MSG_H4 MSG_MOVE_E1 MSG_MOVE_E2 MSG_MOVE_E3 MSG_MOVE_E4 MSG_N1 MSG_N2 MSG_N3 MSG_N4 MSG_DIAM_E1 MSG_DIAM_E2 MSG_DIAM_E3 MSG_DIAM_E4 MSG_E1 MSG_E2 MSG_E3 MSG_E4`